### PR TITLE
Update the mobile view of market details

### LINKED ISSUE #2258: UI Refresh: Access full market details from mobile
### Description

In the latest mobile design the market details have been reduced to display only some data points. There should be a way to expand and show all the market info.

### Solution

Make the markets details bar expandable on mobile to show full info.

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
Show all the market info on the mobile following the [design](https://www.figma.com/file/YWU3PMsnqZaqr1gABgAFJK/UI-Refresh?type=design&node-id=9276-60858&t=pW7PNxmIGtBYqo5W-4)
image

## Related issue
closes  #2258 

## Screenshots (if appropriate):
image
image

### CODE REVIEW

File: sections/futures/MarketDetails/MarketDetails.tsx [hunk 1]
```diff
@@ -190,24 +200,47 @@ const MarketSkew: React.FC<MarketDetailsProps> = memo(({ mobile }) => {
...
-			}
+			mobile={mobile}
+			dataKey={MarketDataKey[dataKey]}
+			value={mobile ? mobileValue : desktopValue}
 		/>
 	);
 });
 
+const OpenInterestLongDetail: React.FC<MarketDetailsProps> = (props) => (
+	<OpenInterestDetail isLong {...props} />
```
Comment (avclarke) [inline sections/futures/MarketDetails/MarketDetails.tsx:238]: Seems a bit overkill to have two separate components for a hardcoded prop, should we just pass the isLong prop at the point these are rendered instead, so we only use a OpenInterestDetail component?

File: sections/futures/MarketDetails/MarketDetails.tsx [hunk 2]
```diff
@@ -35,22 +37,45 @@ type MarketDetailsProps = {
...
+	);
+
+	const MarketDetailsMobile = () => (
+		<MarketDetailsContainer mobile={mobile}>
+			<IndexPriceDetail mobile={mobile} />
+			<MarketSkew mobile={mobile} />
+			<HourlyFundingDetail mobile={mobile} />
+			<MarketPriceDetail mobile={mobile} />
+			<DailyChangeDetail mobile={mobile} />
+			<FlexDivRow style={{ columnGap: '25px' }}>
```
Comment (avclarke) [inline sections/futures/MarketDetails/MarketDetails.tsx:67]: Prefer to avoid inline styles, could we add this as a prop to FlexDivRow instead maybe?

**Review by avclarke — COMMENTED** [review]
Looks good, just a couple of minor clean up things which could be good to include.

File: sections/futures/MarketDetails/MarketDetails.tsx [hunk 3]
```diff
@@ -35,22 +37,45 @@ type MarketDetailsProps = {
...
 
+interface OpenInterestDetailProps extends MarketDetailsProps {
+	isLong: boolean;
+}
+
 const MarketDetails: React.FC<MarketDetailsProps> = ({ mobile }) => {
 	const dispatch = useAppDispatch();
 	const showHistory = useAppSelector(selectShowHistory);
+
+	const MarketDetailsDesktop = () => (
```
Comment (avclarke) [inline sections/futures/MarketDetails/MarketDetails.tsx:48]: Would be good to render these conditionally, as currently a waste rendering both but only using one currently

File: sections/futures/MarketDetails/MarketDetails.tsx (hunk 1 above)
Comment (LeifuChen) [reply sections/futures/MarketDetails/MarketDetails.tsx:238]: Good idea. Thanks.

File: sections/futures/MarketDetails/MarketDetails.tsx (hunk 2 above)
Comment (LeifuChen) [reply sections/futures/MarketDetails/MarketDetails.tsx:67]: Yes. That's a good idea. Updated.

**Review by avclarke — APPROVED** [review]
Changes look good!

**Comment by platschi** [comment]
Screenshot 2023-05-17 at 11 36 33

What if we add a space between OI(L) and (S), as in `OI (L)` and `OI (S)` for better readability?

**Review by platschi — APPROVED** [review]

**Pull request merged**